### ISSUE TITLE
Does not work for windows

### ISSUE THREAD

**Issue by hqjs** [hqjs/vscode-hq-live-server#3]
Could not locate active project in windows.

**Comment by mmcintyre123** [comment]
Yes, please note this in the project readme until it can be enabled on Windows.  Also tried it in VSCode's WSL: Ubuntu mode and it doesn't work on that either, though the Go Live button at least appears.

**Comment by hqjs** [comment]
There should be an easy to fix problem with vscode project structure on windows, but I don't have any windows machine to reproduce and fix it. Some help would be appreciated.

**Comment by mopsek** [comment]
@hqjs  I have VirtualBox with Windows 10 on board. Could possibly help you guys with it.

**Comment by hqjs** [comment]
That would be great! PRs are welcome or even just some debugging will be
extremely helpful.

**Comment by shba007** [comment]
I have a windows machine I am looking forward to use the extension.

**Comment by hqjs** [comment]
@mopsek can you please assist with debugging?

**Comment by shba007** [comment]
@hqjs if you need any help i will be happy to help

**Comment by hqjs** [comment]
@Shirsendu-Bairagi thanks a lot! I guess there are two problems here:
1. `hq` server itself is not very accurate while working with paths (using `/` instead of `path.separator`)
2. seems like vscode on windows might have a bit different project organization

I would be glad to fix all problems if only I know where to look so any debuggin information would be really helpful.

**Comment by shba007** [comment]
Ok i try to clone and debug your project....when i complete i will directly send it to @hqjs email.

**Comment by hqjs** [comment]
@mopsek the problem seems to be here https://github.com/hqjs/vscode-hq-live-server/blob/ac466cf739b3b4f8f6d8c7c2be31b22f67f3b1ca/project-resolver.js#L8

Can you please provide some debug values for
```js
workspaceFolders.map(project => `${project.uri.fsPath}/`);
activeFileName;
```

**Comment by mopsek** [comment]
> @mopsek the problem seems to be here
> 
> https://github.com/hqjs/vscode-hq-live-server/blob/ac466cf739b3b4f8f6d8c7c2be31b22f67f3b1ca/project-resolver.js#L8
> 
> Can you please provide some debug values for
> 
> ```js
> workspaceFolders.map(project => `${project.uri.fsPath}/`);
> activeFileName;
> ```

Sure.

**Comment by mopsek** [comment]
Fixed here: https://github.com/hqjs/vscode-hq-live-server/pull/4

**Comment by hqjs** [comment]
Published as v0.0.7
